eng/common/post-build/publish-using-darc.ps1

```diff
@@ -55,15 +55,12 @@ try {
       $optionalParams.Add($SigningValidationAdditionalParameters) | Out-Null
     }
   }
-  # note the custom branch to work around https://github.com/dotnet/arcade/issues/6987 .  Try to keep this rebased off master if publishing changes
```

Comment:
@riarenas would you mind adding the revert of this again? :)

Reply:
Done!